packages/editor/src/components/document-outline/index.js

```diff
@@ -142,7 +142,7 @@ export const DocumentOutline = ( { blocks = [], title, onSelect, isTitleSupporte
 
 export default compose(
 	withSelect( ( select ) => {
-		const { getEditedPostAttribute, getBlocks } = select( 'core/block-editor' );
+		const { getEditedPostAttribute, getBlocks } = select( 'core/editor' );
```

Comment:
`getEditedPostAttribute` is in core/editor but `getBlocks` should still be retrieved from `core/block-editor`

Reply:
Thanks, done!